scipy/stats/_continuous_distns.py

```diff
@@ -5230,6 +5230,42 @@ def func(params, data):
 logistic = logistic_gen(name='logistic')
 
 
+class fermi_dirac_gen(logistic_gen):
+    r"""Fermi-Dirac continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `fermi_dirac` is:
+
+    .. math::
+
+        f(x) = \frac{\exp(-x)}
+                    {(1+\exp(-x))^2}
+
+    In physics it is better known through the survival function used as:
+
+    .. math::
+
+        S(x) = n_F(x) = \frac{1}{exp(x) + 1}
+
+    To use the physicists Fermi-Dirac distribution one should
+    use `fermi_dirac.sf`.
+
+    `fermi_dirac` is a special case of `genlogistic` with ``c=1``.
+
+    %(after_notes)s
+
+    %(example)s
+
+    """
+    pass
+
+
+fermi_dirac = fermi_dirac_gen(name='fermi_dirac')
```

Comment:
Can just `fermi_dirac = logistic_gen(name="fermi_dirac") ` and avoid the extra subclass, it seems.

Reply:
I have made a new PR which should be clearer and not duplicate.